### ISSUE TITLE
Windows OpenGL Application crashes in SetPixelFormat(...) when launched from RenderDoc 1.21

### ISSUE THREAD

**Issue by tlochner-rightware** [baldurk/renderdoc#2691]
## Description

Launching an OpenGL Windows application crashes inside of WinGDI's SetPixelFormat when launched from RenderDoc since upgrading from 1.20 to 1.21.

## Steps to reproduce

Launch the application from RenderDoc v1.21 Launch Application window, the application crashes in SetPixelFormat(...).

When launching the application normally (not using RenderDoc) it launches without issue.
When launching the application from RenderDoc v1.20, it launches without issue.

I unfortunately cannot upload an application or capture for privacy reasons. This bit of code is non-descript enough to share.

```
    PIXELFORMATDESCRIPTOR pfd =
    {
        sizeof(PIXELFORMATDESCRIPTOR),
        1,
        PFD_DRAW_TO_WINDOW | PFD_SUPPORT_OPENGL | PFD_DOUBLEBUFFER,
        PFD_TYPE_RGBA,    
        32,               
        0, 0, 0, 0, 0, 0,
        0,
        0,
        0,
        0, 0, 0, 0,
        0,                
        0,                
        0,                
        PFD_MAIN_PLANE,
        0,
        0, 0, 0
    };

    const auto fmt = ChoosePixelFormat(deviceContext, &pfd);
    SetPixelFormat(deviceContext, fmt, &pfd); // <-- Crashes in here. Read Violation Access memory location 0x20.
```

The crash does not happen in v1.20. I have reverted back to v1.20 as a workaround.

## Environment

* RenderDoc version: 1.21
* Operating System: Windows 10
* Graphics API: OpenGL

**Comment by baldurk** [comment]
Can you test with the latest nightly build? I fixed a bug since v1.21 in this area which might be the same issue.

**Comment by tlochner-rightware** [comment]
Ah! The nightly build launches without issue. Thanks for the prompt reply! I'll stay on 1.20 until the next stable release.
